doc/devel/testing.rst

```diff
@@ -152,12 +152,11 @@ tests it::
    from matplotlib.testing.decorators import image_comparison
    import matplotlib.pyplot as plt
 
-   @image_comparison(baseline_images=['hexbin_empty'], remove_text=True,
+   @image_comparison(baseline_images=['line_dashes'], remove_text=True,
                      extensions=['png'])
-   def test_hexbin_empty():
-       # From #3886: creating hexbin from empty dataset raises ValueError
-       ax = plt.gca()
-       ax.hexbin([], [])
+   def test_line_dashes():
+       fig, ax = plt.subplots()
+       ax.plot(range(10), linestyle=(0, (3, 3)), lw=5)
 
 The first time this test is run, there will be no baseline image to compare
 against, so the test will fail.  Copy the output images (in this case
```

Comment:
@timhoffm the paths below need to be updated as well

Reply:
Done.